### PR TITLE
LPD-26643 Address autoselect during checkout not working for account defaults and reorder

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-26643

### CODE REVIEW

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

File: modules/test/playwright/pages/commerce/commerceAdminOrderDetailsPage.ts [hunk 1]
```diff
@@ -8,20 +8,28 @@ import {Locator, Page} from '@playwright/test';
...
 	readonly page: Page;
+	readonly reorderButton: Locator;
 
 	constructor(page: Page) {
 		super(
 			page,
 			'#_com_liferay_commerce_order_web_internal_portlet_CommerceOrderPortlet_editOrderContainer .dnd-table'
 		);
-
+		this.checkoutButton = page.getByRole('button', {name: 'Checkout'});
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminOrderDetailsPage.ts:22]: Add `exact: true` property

File: modules/test/playwright/pages/commerce/commerceAdminOrderDetailsPage.ts [hunk 2]
```diff
@@ -8,20 +8,28 @@ import {Locator, Page} from '@playwright/test';
...
 		this.commerceOrderAccountEntryName = page.getByTestId(
 			'commerceOrderAccountEntryName'
 		);
 		this.headerDetailsTitle = page.getByTestId('headerDetailsTitle');
 		this.page = page;
+		this.reorderButton = page.getByRole('button', {name: 'Reorder'});
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminOrderDetailsPage.ts:28]: Same here

File: modules/test/playwright/pages/commerce/commerceAdminOrderDetailsPage.ts [hunk 3]
```diff
@@ -8,20 +8,28 @@ import {Locator, Page} from '@playwright/test';
...
+	}
+
+	async redordeProduct() {
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminOrderDetailsPage.ts:31]: Please fix typo here

File: modules/test/playwright/tests/commerce/commerce-order-content-web/placedOrder.spec.ts [hunk 4]
```diff
@@ -137,3 +139,144 @@ test('LPD-25831 Placed orders widget configuration to display full addresses and
 	await expect(placedOrdersPage.shippingAddress).toContainText('Alabama');
 	await expect(placedOrdersPage.shippingAddress).toContainText(phoneNumber);
 });
+
+test('LPD-26643 Address autoselect during checkout not working for account defaults and reorder', async ({
```
Comment (alessiorendina) [inline modules/test/playwright/tests/commerce/commerce-order-content-web/placedOrder.spec.ts:143]: Describe what the test does instead

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@monaco-fabio please check my comments, thank you!

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:stop

**Comment by monaco-fabio** [comment]
#5119

**Pull request closed without merging**